mantidproject/mantid
# Load separate period good frame values

Fixes #7820

LoadMuonNexus fails to load data for good frames by period. At present the `goodfrm` log is read from `/run/instrument/beam/frames_good` and it is wrong as is it a single value which is the sum of all of the periods. It should be read from `/run/instrument/beam/frames_period_daq/`, which contains a vector of values for each period.

Tester: 
1. Check a multi-period dataset: Open MUSR00015189.nxs in Mantid. Right-click on both workspaces to show "Sample Logs". Check `goodfrm`, it should be 2500 in both cases. If you open the file in HDFView, these numbers should match what you see in /run/instrument/beam/frames_period_daq/.
2. Check a single-period dataset: Open EMU00006473.nxs and check that `goodfrm` matches what you see in `/run/instrument/beam/frames_good` using HDFView.

Release notes changes: http://www.mantidproject.org/index.php?title=Release_Notes_3_5_MuonAnalysis&diff=24301&oldid=24294
